### PR TITLE
adding Doumentation

### PR DESCRIPTION
**Pull request by laserlab**
# title 

I made a useful comment that should go in the main branch!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,7 @@
-# CHSH
+# Bell Inequality on a Quantum computer
```
Comment (admarga) [inline README.md:1]: What is a bell inequality?

**Review by admarga — CHANGES_REQUESTED** [review]
Could use more examples.

File: README.md (hunk 1 above)
Comment (laserlab) [reply README.md:1]: (https://en.wikipedia.org/wiki/CHSH_inequality)

**Comment by laserlab** [comment]
Added more examples

**Pull request merged**